### DISCUSSION TITLE
type error on $route.params.id

### DISCUSSION THREAD

**Discussion by AtharFazle** [posva/unplugin-vue-router#462] [Questions and Help] [answered]
Property 'id' does not exist on type 'Record<never, never> | { path: string; } | { id: string; }'.
Property 'id' does not exist on type 'Record<never, never>'.

![Uploading Screenshot 2024-07-31 165249.png…]()

**Comment by posva** [comment, marked as answer]
See https://github.com/posva/unplugin-vue-router/issues/459#issuecomment-2258619918
Please, search issues before opening new ones 🙏
